modules/android/pages/vector-search.adoc

```diff
@@ -0,0 +1,95 @@
+= Vector Search
+:page-aliases: 
+ifdef::show_edition[:page-edition: {release}]
+ifdef::prerelease[:page-status: {prerelease}]
+:page-role:
+:description: Use Vector Search for AI applications.
```

Comment:
Same note about actually using the {description} in your document.

Reply:
Done!